src/main/java/com/example/java/test/junior/developer/security/KeycloakRestClient.java

```diff
@@ -0,0 +1,78 @@
+package com.example.java.test.junior.developer.security;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+import org.springframework.web.server.ResponseStatusException;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+import reactor.core.publisher.Mono;
+
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class KeycloakRestClient {
```

Comment:
Why have you created a new client? We already have `KeycloakAuthClient`. Add method there.